aten/CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ else()
   project(ATen CXX C)
 endif()
 
+IF(WITH_ROCM)
+  INCLUDE(${CMAKE_CURRENT_SOURCE_DIR}/cmake/LoadHIP.cmake)
```

Comment:
Note for a follow up PR: we're moving all of the cmake modules into the top-level cmake directory, so LoadHIP should live there similarly.